From 97c80e875d9a0d77fac292f892dc91ac909feb0a Mon Sep 17 00:00:00 2001
From: Steven Dake <sdake@redhat.com>
Date: Mon, 24 Feb 2014 10:12:02 -0700
Subject: [PATCH] Remove pbr runtime dependency and replace with build time
 calculation

Requires some sed operation on REDHATOSAPPLYCONFIGVERSION and
REDHATOSAPPLYCONFIGRELEASE to runtime modify the package.
---
 os_apply_config/version.py | 13 +++++++++++--
 1 file changed, 11 insertions(+), 2 deletions(-)

diff --git a/os_apply_config/version.py b/os_apply_config/version.py
index d4af00b..fc30ea5 100644
--- a/os_apply_config/version.py
+++ b/os_apply_config/version.py
@@ -13,6 +13,15 @@
 #    under the License.
 
 
-import pbr.version
+class VersionInfo(object):
+    release = "REDHATOAPPLCONFIGRELEASE"
+    version = "REDHATOSAPPLYCONFIGVERSION"
 
-version_info = pbr.version.VersionInfo('os-apply-config')
+    def version_string(self):
+        return self.version
+
+    def release_string(self):
+        return self.release
+
+version_info = VersionInfo()
+version_string = version_info.version_string
